App/Components/HomeScreen.js

```diff
@@ -0,0 +1,61 @@
+import React, {Component} from 'react';
+import {View, Text, StyleSheet,FlatList,TouchableOpacity} from 'react-native';
+
+
+const style = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#5A6978',
+  },
+  countryContainer: {flex: 0.15, marginStart:20,marginEnd:20, backgroundColor: '#5A6978'},
+  countryBrand: {
+    width: 72,
+    height: 55,
+    marginTop: 15,
+    marginStart: 20,
+    backgroundColor: '#13CE66',
+  },
+  countryName: {
+    position: 'absolute',
+    marginTop: 25,
+    marginLeft: 130,
+    fontSize: 20,
+    color: 'white',
+  },
+});
+
+const list = (item) => {
```

Comment:
Renomear para algo mais claro. Por exemplo, `renderCountryListItem`

Reply:
refactored to countryItemRenderFlatList